src/App.css

```diff
@@ -28,7 +28,7 @@
 
 @media (prefers-reduced-motion: no-preference) {
   a:nth-of-type(2) .logo {
-    animation: logo-spin infinite 20s linear;
+    animation: logo-spin infinite 1s linear;
```

Comment:
This is too fast for the user. Lets stick to 5s or more.

Reply:
Changes made.